### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by FredTheDino**

### CODE REVIEW

File: pygame.docs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# pg.Color(r, g, b)
+
+Creates a new color with the given
+r=red, g=green, b=blue, a=opacity values.
```
Comment (TheZoq2) [inline pygame.docs]: a is mentioned but not part of the function signature

File: ribs.py [hunk 2]
```diff
@@ -68,7 +68,7 @@ def _to_keycode(key):
 def key_down(key):
     """
         Takes a key, that's either a keycode or a character,
-        and says if it's down or not
+        and says if it's down or not.
```
Comment (TheZoq2) [inline ribs.py]: Might be clearer to say "returns true if the corresponding key is pressed"

File: ribs.docs [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+A super small and easy to use collection of functions
+to help with some things pygame don't help with.
+
+Also note that not everything is documented, if it's not
+documented you probably don't have to care about it.
+
+
+# key_down(key)
+Takes a key, that's either a keycode or a character,
+and says if it's down or not.
```
Comment (sornas) [inline ribs.docs:11]: Might want to show both a keycode and a character being used. Arrow keys for example.

File: ribs.docs [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    print("COWABUNGA")
+--!
+This code would print "COWABUNGA" to the console when the "a" button is held.
+
+## ex
+!--
+if key_down(pg.K_LEFT):
+    print("Move left")
+--!
+This code would print "Move left" to the console when the spacebar is held.
```
Comment (sornas) [inline ribs.docs:25]:
```suggestion
This code would print "Move left" to the console when the left arrow key is held.
```

**Pull request merged**